### ISSUE TITLE
Remediation scan using FodUploader.exe

### ISSUE THREAD

**Issue by abelbrazal** [ryancblack/FoDUploader#16]
There is no option to submit a remediation scan using FodUploader.exe.

**Comment by mtgibbs** [comment]
Yeah sorry.  We don't really support this uploader anymore.  We'll accept any PRs anyone wants to submit to fix this problem though.

**Comment by mtgibbs** [comment]
@cripher Is there any help you might need with our more supported uploader .jar?  We make sure to keep that one up to date.

**Comment by abelbrazal** [comment]
@mtgibbs Looks like they are okay with the uploader.jar. They just notice that the remediation feature is missing in exe.
